src/lib/utils/timer.cpp

```diff
@@ -84,7 +84,7 @@ std::string Timer::result_string_bps() const
    {
    const size_t MiB = 1024 * 1024;
 
-   const double MiB_total = static_cast<double>(events()) / MiB;
+   const double MiB_total = std::bit_cast<double>(events()) / MiB;
```

Comment:
This (and the similar cases in this file) seems wrong to me - we want an integer value converted to a double value, not a double with the bit pattern of the integer